hotkey.go

```diff
@@ -44,6 +44,20 @@ import (
 	"runtime"
 )
 
+var (
+	errIn  chan<- Event
+	errOut <-chan Event
+)
+
+// Err returns a channel that receives a signal when an unexpected error has occured.
+// On Linux systems using X11 this indicates a failed XGrabKey.
+// Not used for Windows and Darwin
+func Err() <-chan Event { return errOut }
```

Comment:
It is the only possible way to indicate to the user that a key binding won't work and there is no way to make the API consistent with the X11 async error behaviour. I don't personally need this type of notification. Just though from the perspective of the package, this is an error that the lib is aware and someone could find useful in case they want to notify the user that there is a key binding issue.

Totaly agree that it is inconsistent, it is confusing from an API perspective and in the end it even is not that helpful...
If this was my repo and you were making this PR, I would have been against it also 😄

So glad that we agree on that point. Will remove that part and keep it as silent handling.